airbyte-webapp/src/views/Connection/CatalogDiffModal/components/DiffFieldTable.module.scss

```diff
@@ -2,7 +2,7 @@
 @use "./DiffSection.module.scss";
 
 .accordionSubHeader {
-  @extend .sectionSubHeader;
+  @extend %sectionSubHeader;
```

Comment:
Can revert, but linter wanted us to use placeholders over classes as classes can result in unintended side effects.